lib/dry/cli/inline.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Dry
+  class CLI
+    require 'dry/cli'
+
+    module Inline
+      extend Forwardable
+
+      AnonymousCommand = Class.new(Dry::CLI::Command)
+
+      delegate %i[desc example argument option] => AnonymousCommand
+
+      def run(arguments: ARGV, out: $stdout)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused method argument - `out`. You can also write as `run(*)` if you want the method to accept any arguments but don't care about them.](https://app.codacy.com/gh/dry-rb/dry-cli/pullRequest?prid=4800973)